packages/Graph/Actions/User/GetDirectReports.cs

```diff
@@ -80,10 +87,19 @@ protected override void PopulateParameters(DialogStateManager state, Dictionary<
                 maxCount = this.MaxCount.GetValue(state);
             }
 
+            // Select minimum of the "id" field from the object
+            string fieldsToSelect = DefaultIdField;
+
+            if (this.FieldsToSelect != null)
```

Comment:
This should then change to handling if the array is null or empty, then add the ID property name.

Reply:
Sure.